### PR TITLE
Support custom messages (per project)

### PR DESCRIPTION
**Pull request by TBonnin**
Implementation of https://github.com/guardian/prout/issues/52 

Allow devs to specify for each checkpoint a set of files (one per status) which content would be appended to the PR comment.

```
{
  "url": "https://www.theguardian.com",
  "overdue": "15M",
  "messages": {
    "seen": "prout/seen.md",
    "overdue": "prout/overdue.md"
  }
}
``` 

screen shot 2017-06-08 at 16 59 25

### CODE REVIEW

File: test/lib/Helpers.scala [hunk 1]
```diff
@@ -38,6 +38,17 @@ trait Helpers extends PlaySpec with OneAppPerSuite with Inspectors with ScalaFut
...
 
+  def lastCommentOn(pr: PullRequest): String =
+    pr
+      .comments2
+      .list()
+      .all()
+      .futureValue
+      .lastOption
+      .map(_.body)
+      .getOrElse("")
```
Comment (rtyley) [inline test/lib/Helpers.scala]: I often use `.mkString` instead of `.getOrElse("")`, as they're equivalent.

**Comment by TBonnin** [comment]
@rtyley I have added the custom messages info to the config diagnostic page: 855e471

screen shot 2017-06-12 at 10 56 14

Let me know what you think. Thanks

**Comment by rtyley** [comment]
Here's [the first comment](https://github.com/guardian/frontend/pull/17107#issuecomment-308081899) to use this new help:

image

**Comment by TBonnin** [comment]
I noticed a small issue with the diagnostic page: 
screen shot 2017-06-13 at 13 59 19
https://prout-bot.herokuapp.com/view/guardian/frontend

I will push a fix when I can.

**Pull request merged**